### PR TITLE
Order certificate with email validation

### PR DESCRIPTION
**Pull request by andregomesb**
Add GetDVApproverList and DVOrder to allow ordering certificate using email validation

### CODE REVIEW

File: lib/global_sign/dv_order/request.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          OrderRequestParameter: order_request_paramater,
+          OrderID: @order_id,
+          ApproverEmail: @approver_email,
+          ContactInfo: contact_info
+        }
+      end
+
+      private
+
+      def order_request_paramater
```
Comment (ravelll) [inline lib/global_sign/dv_order/request.rb]: Is `paramater` a typo for `parameter`?

Comment (andregomesb) [inline lib/global_sign/dv_order/request.rb]: Yes, it is. Fixed.

File: lib/global_sign/dv_order/request.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
+module GlobalSign
+  module DVOrder
+    class Request < GlobalSign::Request
+      KIND_RENEWAL = 'renewal'.freeze
+
+      def initialize(product_code:, order_kind:, validity_period_months:,
+                     csr:, approver_email:, order_id:,
+                     renewal_target_order_id: nil, contract_info: nil)
```
Comment (ku00) [inline lib/global_sign/dv_order/request.rb]: Please remove newlines of this method arguments if you don't need them.

File: lib/global_sign/get_dv_approver_list/response.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+module GlobalSign
+  module GetDVApproverList
```
Comment (ku00) [inline lib/global_sign/get_dv_approver_list/response.rb]: Please rename this module. It is noun, such as `OrderGetterByOrderId`.

File: lib/global_sign/dv_order/request.rb (hunk 2 above)
Comment (andregomesb) [inline lib/global_sign/dv_order/request.rb]: Changed to single line

File: lib/global_sign/get_dv_approver_list/response.rb (hunk 3 above)
Comment (andregomesb) [inline lib/global_sign/get_dv_approver_list/response.rb]: Changed to `DVApproverList`

**Pull request merged**